app/assets/javascripts/price_policy.js

```diff
@@ -0,0 +1,127 @@
+    $adjustmentFields.val(isChecked ? "1" : "0");
+    hardToggleField($adjustmentFields.siblings(".js--cancellationCost").filter(":input"), isChecked);
+    // Show/hide the pricing spans
+    return $adjustmentFields.siblings(".js--cancellationCost").filter("span").toggle(!isChecked);
+  };
+
+  $(".js--canPurchase").change(evt => toggleFieldsInSameRow($(evt.target))).trigger("change");
+
+  $(".js--fullCancellationCost").change(function(evt) {
+    const $elem = $(evt.target);
+    toggleFullCancellationCostInCurrentCell($elem);
+    if ($elem.parents(".js--masterInternalRow").length) {
+      toggleFullCancellationInAdjustmentRows($elem.is(":checked"));
+      // Update trigger the adjustment rows to be updated off of the value
+      if (!$elem.is(":checked")) { return $elem.parents(".js--cancellationCostContainer").find(".js--cancellationCost").trigger("keyup"); }
+    }
+  }).trigger("change");
+
+  $(".js--masterInternalRow input[type=text]").keyup(evt => updateAdjustmentFields($(evt.target))).trigger("keyup");
+
+  $(".js--price-policy-note-select").on("change", function(event) {
+    const selectedOption = event.target.options[event.target.selectedIndex];
+    const noteTextField = $(".js--price-policy-note");
+    if (selectedOption.value === "Other") {
+      return noteTextField.attr("hidden", false).val("");
+    } else {
+      noteTextField.attr("hidden", true);
+      return noteTextField.val(selectedOption.value);
+    }
+  });
+
+  $(".js--minDuration").on("change", function (event) {
+    setMinDurationHours(event.target)
+    preventDuplicateMinDurations()
+  });
+
+  $(".js--minDuration").each(function (index, element) {
+    setMinDurationHours(element)
+  });
+
+  function setMinDurationHours(params) {
+    const columnIndex = params.dataset.index;
+    const minDuration = params.value;
+    $(`input[name*='duration_rates_attributes][${columnIndex}][min_duration_hours]']`).val(minDuration);
+  }
+
+  function preventDuplicateMinDurations(params) {
+    var found = [];
+    var duplicate = [];
+    $(".js--minDuration").each(function (idx, ele) {
+      var minDurationValue = $(ele).val();
+      if (found.includes(minDurationValue)) {
+        duplicate.push(minDurationValue)
+      } else if (minDurationValue) {
+        found.push(minDurationValue);
+      }
+    });
+
+    if (duplicate.length > 0) {
+      var dupes = $(".js--minDuration").filter(function (i, element) {
+        return $(element).val() == duplicate[0] // since there are only 3 inputs, there can only be 1 duplicate value
+      })
+      // reset all borders
+      $(".js--minDuration").each(function (i, element) {
+        $(element).css("border", "1px solid #ccc");
+      })
+      // set border to red for duplicate values
+      dupes.each(function (i, element) {
+        $(element).css("border", "2px solid red");
+      })
+      // disable submit button
+      $("input[type=submit]").attr("disabled", "disabled");
```

Comment:
I believe [`.prop("disabled", true)`](https://api.jquery.com/prop/#prop-propertyName-value) is the preferred mechanism for setting boolean properties.